Decred Wallet/Extensions/UIButton.swift

```diff
@@ -54,4 +53,22 @@ extension UIButton {
             self.setNeedsLayout()
         }
     }
+
+    private func imageWithColor(color: UIColor) -> UIImage? {
+        let rect = CGRect(x: 0.0, y: 0.0, width: 1.0, height: 1.0)
+        UIGraphicsBeginImageContext(rect.size)
+        let context = UIGraphicsGetCurrentContext()
+        
+        context?.setFillColor(color.cgColor)
+        context?.fill(rect)
+        
+        let image = UIGraphicsGetImageFromCurrentImageContext()
+        UIGraphicsEndImageContext()
+        
+        return image
+    }
+    
+    func setBackgroundColor(_ color: UIColor, for state: UIControl.State) {
+        self.setBackgroundImage(imageWithColor(color: color), for: state)
```

Comment:
You should be able to set background colors for different states from IB. This necessary?